tests/sentry/backup/test_coverage.py

```diff
@@ -61,8 +60,6 @@ def test_exportable_final_derivations_of_sentry_model_are_collision_tested():
             if necessitates_collision_test:
                 want_collision_tested.add(model_relations.model)
 
-    # TODO(actor): Remove after actor refactor completed.
```

Comment:
I think this might be backwards? We can remove the check for `Team`, since it now no longer has any qualifying `uniques` sets post-`actor` removal. But `actor` can still references `user_id` and must therefore be tested for collisions, so we should keep it (and the comment) around.

Reply:
Ah that makes sense. In that case maybe you can remove both? If not, this should be fine, though we should still keep the `TODO` if we leave at least one line.